Automattic/wp-calypso
# Big Sky entry point

Entry point for Big Sky in the onboarding flow in staging environment. 

- [ ] Image/animation for the card (to do in follow-up)
- [ ] Finalize copy (to do in follow-up)
- [x] Remove the step for inputting the prompt. Just shows loading page. 
- [x] Hide on narrow viewports
- [x] Hide on certain plans 
- [x] Only show with certain goals
- [x] Check if homepage exists before creating one
- [x] Handle browser navigation: If navigating "back" to the loading page, it should actually go to the previous step of design-choices. 

## Proposed Changes

Entry point for Big Sky in the onboarding flow. 

It adds a new card to the /design-choices step, if the conditions of the new `useIsBigSkyEligible()` are met, which are: 
- On a premium (explorer) plan
- Have selected specific goals of `promote` or `other`. 
- is the config enabled (not yet in production)
- Is the viewport wide enough (temporary workaround to UX issues in Big Sky on narrow views)

Follow-ups will be deployed for: 
- Finalized copy for the design choice 
- Finalized image for the design choice card 

## Testing Instructions



Purchase an explorer plan
- can start at http://calypso.localhost:3000/start/premium/
- Choose a domain
- Purchase the plan (explorer)
- You should see the Big Sky card in design-choices, click it
- You should see a loading page, which then takes you to the site editor
- Big Sky should appear

Also test: 
- Go "back" after landing in Big Sky. It should take you back to design-choices
- It should not show on any other plans (only explorer) 

You can also go straight to `http://calypso.localhost:3000/setup/site-setup/design-choices?siteSlug=SITESLUG` to target a specific test site. 

## Pre-merge Checklist
